build.gradle.kts

```diff
@@ -19,22 +17,16 @@ buildscript {
         maven { url = uri("https://dl.bintray.com/icerockdev/plugins") }
     }
     dependencies {
-        with(Deps.Plugins) {
-            listOf(
-                androidApplication,
-                androidLibrary,
-                kotlinMultiplatform,
-                kotlinKapt,
-                kotlinAndroid
-            )
-        }.let { plugins(it) }
+        plugin(Deps.Plugins.androidApplication)
+        plugin(Deps.Plugins.androidLibrary)
+        plugin(Deps.Plugins.kotlinMultiplatform)
+        plugin(Deps.Plugins.kotlinKapt)
+        plugin(Deps.Plugins.kotlinAndroid)
     }
 }
```

Comment:
all buildscript block not required - please remove it, because all added dependencies already in classpath because connected in buildSrc